.github/workflows/ci.yml

```diff
@@ -0,0 +1,80 @@
+name: CI
```

Comment:
This name change will break our build badge in the README.md. Can we update that?

Reply:
@scalvert good point! Will update

Reply:
@scalvert updated!

Reply:
Thank you!